WorkloadManagementSystem/Agent/JobAgent.py

```diff
@@ -341,7 +341,6 @@ def execute(self):
       result = self._submitJob(jobID, params, ceDict, optimizerParams, proxyChain,
                                processors, wholeNode, maxNumberOfProcessors, mpTag)
       if not result['OK']:
-        self.__report(jobID, 'Failed', result['Message'])
```

Comment:
I think, this never happens, so we do not have problems with it. But if it happens, the user job should be rescheduled I think since it is not likely that the user application is the culprit